config/config.go

```diff
@@ -922,6 +922,9 @@ type ConsensusConfig struct {
 	CreateEmptyBlocks         bool          `mapstructure:"create_empty_blocks"`
 	CreateEmptyBlocksInterval time.Duration `mapstructure:"create_empty_blocks_interval"`
 
+	// Max transactions per block. No limit if <= 0.
```

Comment:
This comment is likely to be misleading. This property limits the maximum number when CREATING a block, and the block suggested by peer may be larger than this.

Reply:
May we change it to

// Max transactions per block when creating a block. No limit if <= 0.

In case you're interested in putting the variable into Consensus.BlockParams, refer to https://github.com/iproudhon/ostracon.wip/commit/c7bcf501040f3601da4b29fc8fb743e3d400c6d8

Reply:
Updated comment to indicate that it's not a global configuration variable.